### DISCUSSION TITLE
"script:ld+json" in Meta_V2 doesn't render

### DISCUSSION THREAD

**Discussion by adamgiebl** [remix-run/remix#6182] [Q&A]
### What version of Remix are you using?

1.15.0

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

Follow [this](https://remix.run/docs/en/main/route/meta-v2 ) documentation to add "script:ld+json" as a part of your meta tags.
This is what it should look like based on the documentation:
```
export const meta: V2_MetaFunction = () => {
  return [
    {
      "script:ld+json": {
        "@context": "https://schema.org",
        "@type": "Organization",
        name: "Remix",
        url: "https://remix.run",
      },
    },
  ];
};
```

### Expected Behavior

It should render a <script> tag with this structured json data inside.

### Actual Behavior

This doesn't render a script tag in your <head> section with the structured json data.

**Comment by adamgiebl** [comment]
And yes, I do have the `v2_meta: true` future flag enabled, every other meta tag works fine.

**Comment by kiliman** [comment]
I just tried it and it works fine for me.

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/47168/233187184-b74152a5-ca18-410d-bd4d-6f82814411b8.png">

https://stackblitz.com/edit/remix-run-remix-kx8vdy?file=app/routes/_index.tsx

**Comment by adamgiebl** [comment]
Seriously? I tried on two completely different Remix projects and didn't work on either. Are you on the same version?

**Comment by kiliman** [comment]
Remix v1.15.0

Did you check out the stackblitz?

<img width="320" alt="image" src="https://user-images.githubusercontent.com/47168/233189373-86311e64-025d-4f64-bf21-720052de47ea.png">
